microsoft-onedrive/1.0/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,53 @@
+= Microsoft OneDrive Connector 1.0 - Mule 4
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for Microsoft OneDrive (OneDrive Connector) enables you to access Microsoft Graph APIs to provide a unified programmability model that you can use to access the tremendous amount of data in Microsoft 365, Windows 10, and Enterprise Mobility + Security.
+
+For compatibility information and fixed issues, see the Microsoft OneDrive Connector Release Notes.
+
+== Prerequisites
+
+To use this connector, you must be familiar with:
+
+* Microsoft Graph API 1.0
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements and global elements in a Mule flow
+* Creating a Mule app using Anypoint Studio (Studio)
+
+Before creating an app, you must have:
+
+* Credentials to access the target resource
+* Anypoint Studio version 7.3.5 or later
+
+== Common Use Cases for the Connector
+
+Some common use cases for OneDrive Connector include:
+
+* Sales Operations
+** When a sales contract comes in through the Salesforce CRM system, upload the contract to OneDrive and collaboratively review the contract with the Deal Desk. You can assign approval tasks, which when approved or rejected, notifies the contract owner.
```

Comment:
notifies > notify

Suggestion: You can assign approval tasks to notify the contract owner when a contract is approved or rejected.